src/__tests__/Cloud-test.js

```diff
@@ -55,7 +55,11 @@ describe('Cloud', () => {
   it('run passes options', () => {
     Cloud.run('myfunction', {}, { useMasterKey: false });
 
-    expect(CoreManager.getCloudController().run.mock.calls[0]).toEqual(['myfunction', {}, {}]);
+    expect(CoreManager.getCloudController().run.mock.calls[0]).toEqual([
+      'myfunction',
+      {},
+      { useMasterKey: false }
```

Comment:
Why does it invoke with `{ useMasterKey: false }` instead of omitting it as it did before, which implies `false`? Is that also sent to the server?

Reply:
`useMasterKey: false` should have been sent to the server, this test was wrong.
```
if(options.useMasterKey)
changed to 
if (Object.hasOwn(options, 'useMasterKey'))
```

Reply:
Which one should be merged first?

Reply:
The bug fix first

Reply:
You mean this PR here first?

Reply:
https://github.com/parse-community/Parse-SDK-JS/pull/2431 This one

Reply:
Merged